Contributing.md

```diff
@@ -30,10 +30,19 @@ This project uses clang-format (stable branch) to check for common style issues.
 
 In most cases, clang-format can and **should** be used to automatically reformat code and solve most formatting issues.
 
-- To run clang-format on all staged files:
+It can be installed using your distro package manager (on Linux) or from [LLVM binary packages](http://releases.llvm.org/download.html) for the latest version.
+
+- To run clang-format on all staged files, use the following command:
   ```
-  git diff --cached --name-only | egrep '[.](cpp|h|mm)$' | xargs clang-format -i
+  git diff --cached --name-only | egrep '[.](cpp|h|mm)$' | xargs clang-format -style=file -i
   ```
+  
+  - If you can't execute clang-format on Windows, try locating `clang-format.exe` in the LLVM installation and place it into the `...Git\usr\bin` directory.
```

Comment:
in the git shell `$PATH`. I'm not sure what kind of path `...Git` is.

Reply:
Oh, that's from my program files installation on windows, I believe C:\Program Files\Git is the default one, but i'm not 100%